include/z64audio.h

```diff
@@ -524,15 +526,15 @@ typedef struct {
     /* 0x03 */ u8 headsetPanRight;
     /* 0x04 */ u8 headsetPanLeft;
     /* 0x05 */ u8 reverbVol;
-    /* 0x06 */ u8 unk_06;
+    /* 0x06 */ u8 harmonicIndexCurAndPrev; // bits 3..2 store curHarmonicIndex, bits 1..0 store prevHarmonicIndex
     /* 0x07 */ u8 unk_07;
     /* 0x08 */ u16 targetVolLeft;
     /* 0x0A */ u16 targetVolRight;
     /* 0x0C */ u16 resamplingRateFixedPoint;
     /* 0x0E */ u16 unk_0E;
     /* 0x10 */ union {
                  SoundFontSound* soundFontSound;
-                 s16* samples; // used for synthetic waves
+                 s16* waveSampleAddr; // used for synthetic waves
```

Comment:
I'm fine with `waveSampleAddr` for now since it pairs nicely with `gWaveSamples`, and if we wanted to change `wave` -> `syntheticWave` everywhere, it should be done at once? Possibly in a separate PR? (Or we could add to this PR)

Reply:
Nah let's bring that up again later